realm/realm-library/src/main/java/io/realm/OrderedCollectionChange.java

```diff
@@ -0,0 +1,34 @@
+package io.realm;
+
+/**
+ * This interface describes the changes made to a collection during the last update.
+ */
+public class OrderedCollectionChange {
+    public long[] getDeletions() { return null; }
+    public long[] getInsertertions() { return null; }
```

Comment:
I'm pretty sure this should be `getInsertions()` and I eerily remember stating that in the RFC too 👅